randovania/games/prime_hunters/pickup_database/pickup-database.json

```diff
@@ -0,0 +1,576 @@
+{
+    "schema_version": 14,
+    "pickup_categories": {
+        "missile": {
+            "long_name": "Missile",
+            "hint_details": [
+                "a ",
+                "missile system"
+            ]
+        },
+        "beam": {
+            "long_name": "Beams",
+            "hint_details": [
+                "a ",
+                "beam"
+            ]
+        },
+        "beam_related": {
+            "long_name": "Beam Related",
+            "hint_details": [
+                "a ",
+                "beam-related upgrade"
+            ],
+            "hidden": true
+        },
+        "missile_related": {
+            "long_name": "Missile Related",
+            "hint_details": [
+                "a ",
+                "missile-related upgrade"
+            ],
+            "hidden": true
+        },
+        "sub_weapon": {
+            "long_name": "Sub-Weapons",
+            "hint_details": [
+                "a ",
+                "Sub-Weapon"
+            ]
+        },
+        "affinity_weapon": {
+            "long_name": "Affinity Weapons",
+            "hint_details": [
+                "an ",
+                "Affinity Weapon"
+            ]
+        },
+        "artifact": {
+            "long_name": "Alimbic Artifact",
+            "hint_details": [
+                "an ",
+                "Alimbic Artifact"
+            ]
+        },
+        "octolith": {
+            "long_name": "Octolith",
+            "hint_details": [
+                "an ",
+                "Octolith"
+            ]
+        },
+        "key": {
+            "long_name": "Key",
+            "hint_details": [
+                "a ",
+                "Key"
+            ]
+        },
+        "major": {
+            "long_name": "Major Item",
+            "hint_details": [
+                "a ",
+                "major item"
+            ],
+            "hidden": true
+        },
+        "energy_tank": {
+            "long_name": "Energy Tanks",
+            "hint_details": [
+                "an ",
+                "Energy Tank"
+            ],
+            "hidden": true
+        },
+        "expansion": {
+            "long_name": "Expansion",
+            "hint_details": [
+                "an ",
+                "expansion"
+            ],
+            "hidden": true
+        }
+    },
+    "generated_pickups": {
+        "Celestial Attameter Artifact": {
```

Comment:
you can actually condense these into just 3 generated pickups:
```json
"Attameter Artifact": {
    "name_template": "{region} Attameter Artifact {i}",
    "gui_category": "artifact",
    "hint_features": [
        "artifact",
        "key"
    ],
    "model_name": "{region}Artifact{i}",
    "offworld_models": {},
    "preferred_location_category": "major",
    "probability_offset": 0.25,
    "progression": [
        "{region}AArtifact{i}",
    ],
    "expected_case_for_describer": "shuffled",
    "extra": {
        "entity_type": 17,
        "artifact_id": 1
    }
}
```
I'll leave a comment in the pool creator showing you how to create the pickups